### PR TITLE
Fix broken link

### LINKED ISSUE #33948: broken link to frameworks
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction

### What specific section or headline is this issue about?

Introducing Express

### What information was incorrect, unhelpful, or incomplete?

the link "Node web frameworks" to https://expressjs.com/en/resources/frameworks.html is broken

### What did you expect to see?

a webpage with frameworks using node

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/server-side/express_nodejs/introduction`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/express_nodejs/introduction/index.md
* Last commit: https://github.com/mdn/content/commit/2528ef17bf90b1822940066c9365641181c5e8e1
* Document last modified: 2024-03-01T02:30:38.000Z

</details>

### PR DESCRIPTION
**Pull request by 3willows**
### Description
 
 Replaced a [broken link](https:/expressjs.com/en/resources/frameworks.html) from 

/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction#introducing_express 

with its [web archive version](https://web.archive.org/web/20240328030121/https:/expressjs.com/en/resources/frameworks.html) 

 
### Motivation 

 Ease of reading / cross referencing 

### Additional details 
 

- [Relevant page ](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction#introducing_express) (search "other popular Node web frameworks") 

- old: https:/expressjs.com/en/resources/frameworks.html
- new: https://web.archive.org/web/20240328030121/https:/expressjs.com/en/resources/frameworks.html 

### Related issues and pull requests 

 
  
None.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks for opening this one. Note that this was reported here:

- [ ] https://github.com/mdn/content/issues/33948

The suggestion is to remove the link instead of replace it, which I think is a good idea here.

File: files/en-us/learn/server-side/express_nodejs/introduction/index.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Other common web-development tasks are not directly supported by Node itself. If
 
 ## Introducing Express
 
-[Express](https://expressjs.com/) is the most popular _Node_ web framework, and is the underlying library for a number of other popular [Node web frameworks](https://expressjs.com/en/resources/frameworks.html). It provides mechanisms to:
+[Express](https://expressjs.com/) is the most popular _Node_ web framework, and is the underlying library for a number of other popular [Node web frameworks](https://web.archive.org/web/20240328030121/https:/expressjs.com/en/resources/frameworks.html). It provides mechanisms to:
```
Comment (bsmth) [inline files/en-us/learn/server-side/express_nodejs/introduction/index.md:94]:
```suggestion
[Express](https://expressjs.com/) is the most popular Node.js web framework, and is the underlying library for a number of other popular Node.js frameworks. It provides mechanisms to:
```

Comment (3willows) [reply files/en-us/learn/server-side/express_nodejs/introduction/index.md:94]: Great idea, apologies for missing the issue!  

As I was reading on I noticed yet another broken link at ["high profile companies" ](https://expressjs.com/en/resources/companies-using-express.html).  

It seems that that page was also removed from Express docs after discussion (see [here](https://github.com/expressjs/expressjs.com/pull/1484), i.e. the same commit as the one removing the frameworks page).

Should I open a similar issue on the lines of [#33948](https://github.com/mdn/content/issues/33948), and refer back to that earlier discussion? 

Thanks for your patience for first time contributors!

Comment (bsmth) [reply files/en-us/learn/server-side/express_nodejs/introduction/index.md:94]: Good catch! I would treat this the same way and make it plain text for that paragraph:

```md
Based on the number of high profile companies that use Express, the number of people contributing to the codebase, and the number of people providing both free and paid for support, then yes, Express is a popular framework!
```

>Thanks for your patience for first time contributors!

Not a problem, we all enjoy collaborating, so it's a pleasure to help out with first time contributions! 🙌🏻

Comment (Josh-Cena) [reply files/en-us/learn/server-side/express_nodejs/introduction/index.md:94]: @3willows You should commit the suggestion above and then make another commit to change the other broken link you mentioned. No need to open an issue or another PR.

Comment (3willows) [reply files/en-us/learn/server-side/express_nodejs/introduction/index.md]: Thanks so much @bsmth and @Josh-Cena, have done so!

**Review by bsmth — APPROVED** [review]
Looking good, thank you!

**Pull request merged**